WordPress/src/main/java/org/wordpress/android/ui/jetpack/backup/details/BackupDownloadDetailsFragment.kt

```diff
@@ -0,0 +1,59 @@
+package org.wordpress.android.ui.jetpack.backup.details
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProvider
+import androidx.lifecycle.ViewModelProviders
+import org.wordpress.android.R
+import org.wordpress.android.WordPress
+import org.wordpress.android.ui.jetpack.backup.BackupDownloadViewModel
+import javax.inject.Inject
+
+class BackupDownloadDetailsFragment : Fragment() {
+    @Inject lateinit var viewModelFactory: ViewModelProvider.Factory
+    private lateinit var parentViewModel: BackupDownloadViewModel
+    private lateinit var viewModel: BackupDownloadDetailsViewModel
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        return inflater.inflate(R.layout.backup_download_details_fragment, container, false)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        val nonNullActivity = requireActivity()
+        (nonNullActivity.application as WordPress).component().inject(this)
+
+        parentViewModel = ViewModelProviders.of(requireActivity(), viewModelFactory)
+                .get(BackupDownloadViewModel::class.java)
+
+        viewModel = ViewModelProviders.of(this, viewModelFactory)
+                .get(BackupDownloadDetailsViewModel::class.java)
```

Comment:
Minor (🔍 ): Consider extracting the `ViewModel` related setup code to it owns private function. Just like you did for the `Activity`.